### PR TITLE
adding levenshtein-search

### PR DESCRIPTION
**Pull request by osamples**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by osamples** [comment]
@dopplershift Hi I have worked with you on two previous PRs. This one is related, but I seem to be getting an error. I was hoping you could maybe help explain the error to me if you've run across it before. This is the[ pypi library](https://pypi.org/project/Levenshtein-search/) I am trying to add to conda-forge.

File: recipes/levenshtein-search/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python
+    - pip
+    - gcc
```
Comment (dopplershift) [inline recipes/levenshtein-search/meta.yaml:21]: Try this:
```suggestion
  build:
    - {{ compiler('c') }}
  host:
    - python
    - pip
```

**Comment by osamples** [comment]
@dopplershift Thank you so much for looking into this! It looked like your solution allowed it to build, but the tests failed. Is there another simple fix for this as well?

File: recipes/levenshtein-search/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - levenshtein_search
```
Comment (dopplershift) [inline recipes/levenshtein-search/meta.yaml:28]: Ugh, it's [capitalized](https://github.com/mattandahalfew/Levenshtein_search#usage):
```suggestion
    - Levenshtein_search
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
These will get this over the finish line I think.

**Comment by osamples** [comment]
@dopplershift It worked!! Thank you so much! This is the best news

**Review by dopplershift — APPROVED** [review]
:tada:

**Pull request merged**